build/images/Dockerfile.build.ubuntu

```diff
@@ -24,7 +24,8 @@ RUN go mod download
 
 COPY . /antrea
 
-RUN make antrea-agent antrea-controller antrea-cni antctl-linux
+RUN make antrea-agent antrea-controller antrea-cni
+RUN CGO_ENABLED=0 make antctl-linux
```

Comment:
it would be good to add a comment along the lines of:

> Disable cgo for antctl in case it is copied outside of the container image (which we sometimes do in tests). It also leads to a smaller binary, and is consistent with how we distribute antctl as part of release assets.

Reply:
Sure, updated.